src/MongoDB.Driver/Encryption/AutoEncryptionOptions.cs

```diff
@@ -148,6 +150,12 @@ public class AutoEncryptionOptions
         /// </value>
         public IReadOnlyDictionary<string, BsonDocument> SchemaMap => _schemaMap;
 
+        internal string ExtraOptionsCsflePath =>
+            (ExtraOptions?.TryGetValue("csflePath", out var csflePathObj) == true && (csflePathObj is string csflePathStr)) ? csflePathStr : null;
+
+        internal bool? ExtraOptionsIsCsfleRequired =>
```

Comment:
I'm not sure that we need to prefix such fields by `ExtraOptions`, it looks like it can be simply IsCsfleRequired

Reply:
Changed.